tests/test_ci/TestControllers.py

```diff
@@ -202,4 +203,94 @@ def test_inform_mailing_list(self, mock_email):
                 '        Lorem Ipsum sit dolor amet...\n        ',
                 'subject': 'GitHub Issue #matejmecka', 'to': 'ccextractor-dev@googlegroups.com'
             }
-)
\ No newline at end of file
+        )
+
+
+    def test_add_blocked_users(self):
+        """
+        Check it will add blocked user
+        """
+        self.create_user_with_role(
+            self.user.name, self.user.email, self.user.password, Role.admin)
+        with self.app.test_client() as c:
+            response = c.post(
+                '/account/login', data=self.create_login_form_data(self.user.email, self.user.password))
+            response = c.post(
+                '/blocked_users', data=dict(user_id=1, comment="Bad user", add=True))
+            self.assertNotEqual(BlockedUsers.query.filter(BlockedUsers.comment=="Bad user").first(),None)
```

Comment:
Better to fetch from blocked users list, the user with id=1 and see if it's not none. Also add other asserts to check the comment and any success message in response.